### PR TITLE
Add metrics to monitor operator balance using Grafana

### PR DESCRIPTION
**Pull request by nohkwak**
## Proposed changes

- Add metrics to monitor childOperatorBalance and parentOperatorBalance of ServiceChain in Grafana. 

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- N/A

## Further comments

-N/A

### CODE REVIEW

File: node/sc/sub_event_handler.go [hunk 1]
```diff
@@ -45,6 +45,24 @@ func (cce *ChildChainEventHandler) HandleChainHeadEvent(block *types.Block) erro
 	logger.Trace("bridgeNode block number", "number", block.Number())
 	cce.handler.LocalChainHeadEvent(block)
 
+	go func() {
```
Comment (2dvorak) [inline node/sc/sub_event_handler.go:48]: Please leave a comment here about the reason for using goroutine to fetch `parentOperatorBalance`.

**Review by hqjang-pepper — COMMENTED** [review]
You need to sign CLA and fix linter problem

File: node/sc/sub_event_handler.go (hunk 1 above)
Comment (nohkwak) [reply node/sc/sub_event_handler.go:52]: Thanks, I added.

**Review by hyunsooda — DISMISSED** [review]
LGTM. Could you fix the formatting the linter pointed out?

**Comment by blukat29** [comment]
To fix for the linter, please run following commands as decribed in https://github.com/klaytn/klaytn/blob/dev/HOW-TO-LINT-YOUR-CHANGE.md.
```bash
# At 'klaytn' directory
gofumpt -w .
goimports -w .
```

**Comment by blukat29** [comment]
@nohkwak Please fix as follows. Apparently `gofumpt` and `goimports` is not enough to satisfy the linter.

```diff
diff --git a/consensus/istanbul/backend/engine.go b/consensus/istanbul/backend/engine.go
index 2bf5a79b..727d66ef 100644
--- a/consensus/istanbul/backend/engine.go
+++ b/consensus/istanbul/backend/engine.go
@@ -28,8 +28,6 @@ import (
        "math/big"
        "time"

-       "github.com/klaytn/klaytn/reward"
-
        lru "github.com/hashicorp/golang-lru"
        "github.com/klaytn/klaytn/blockchain/state"
        "github.com/klaytn/klaytn/blockchain/types"
@@ -43,6 +41,7 @@ import (
        "github.com/klaytn/klaytn/crypto/sha3"
        "github.com/klaytn/klaytn/networks/rpc"
        "github.com/klaytn/klaytn/params"
+       "github.com/klaytn/klaytn/reward"
        "github.com/klaytn/klaytn/rlp"
 )
```

**Review by hqjang-pepper — APPROVED** [review]

**Review by hyunsooda — APPROVED** [review]

File: node/sc/sub_event_handler.go [hunk 2]
```diff
@@ -45,6 +45,28 @@ func (cce *ChildChainEventHandler) HandleChainHeadEvent(block *types.Block) erro
...
 	cce.handler.LocalChainHeadEvent(block)
 
+	// During HandleChainHeadEvent, RPC was sent to inquire the balance of parentOperator,
+	// but RPC send was not processed until chainHeadEvent processing was finished,
+	// and RPC failed due to timeout after event handling was finished.
+	// So, the logic to inquire the parentOperator's balance was handled as a goroutine.
+	go func() {
+		pBalance, err := cce.handler.getParentOperatorBalance()
+		if err != nil {
+			logger.Trace("[SC][View] Failed to get parent operator's balance", "err", err)
```
Comment (henry-will) [inline node/sc/sub_event_handler.go:55]:
```suggestion
			logger.Error("[SC][View] Failed to get parent operator's balance", "err", err)
```
How about change the log level up to Error?

File: node/sc/sub_event_handler.go [hunk 3]
```diff
@@ -45,6 +45,28 @@ func (cce *ChildChainEventHandler) HandleChainHeadEvent(block *types.Block) erro
...
+		} else {
+			logger.Trace("[SC][View] Got parent operator's balance", "parentOperator", cce.handler.GetParentOperatorAddr(), "block", block.Number(), "balance", pBalance)
+			parentOperatorBalanceGauge.Update(pBalance.Int64())
+		}
+	}()
+
+	cBalance, err := cce.handler.getChildOperatorBalance()
+	if err != nil {
+		logger.Trace("[SC][View] Failed to get child operator's balance", "err", err)
```
Comment (henry-will) [inline node/sc/sub_event_handler.go:64]:
```suggestion
		logger.Error("[SC][View] Failed to get child operator's balance", "err", err)
```
How about change the log level up to Error?

**Comment by nohkwak** [comment]
This PR was not merged for a long time, and a build error occurred. 
Since I need to test again and pull request, I'll close it for now.

**Pull request closed without merging**